libs/messenger-bundle/src/KeboolaMessengerBundle.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\MessengerBundle;
+
+use Keboola\MessengerBundle\DependencyInjection\KeboolaMessengerExtension;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
+
+class KeboolaMessengerBundle extends AbstractBundle
```

Comment:
Testem nezarmoutis :)

Reply:
To uz berem trochu daleko, ne? :slightly_smiling_face: Anyways, tohle IMHO dostatecne pokryva funkcni test.

Reply:
Na nejmenovane veci se take spolehalo na funkcni testy a pak to dospelo tam kde to je.
Alespon se ti to dostane hezky pod kuzi a zacnes priste psat testy jako prvni vec :)

Reply:
Nevim moc, na co narazis, ale na tohle je test fakt nesmysl. Pokud na nem trvas, muzem to projit na standupu.